src/__tests__/resources/timeSeriesList.unit.spec.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Cognite AS
+import MockAdapter from 'axios-mock-adapter';
+import { uniqBy } from 'lodash';
+import { CogniteClient } from '../../index';
+import { TimeSeriesList } from '../../resources/classes/timeSeriesList';
+import {
+  DatapointsPostDatapoint,
+  PostTimeSeriesMetadataDTO,
+} from '../../types/types';
+import { randomInt, setupLoggedInClient } from '../testUtils';
+
+describe('TimeSeriesList class unit test', async () => {
+  let axiosMock: MockAdapter;
+  let client: CogniteClient;
+  let createdTimeSeries: TimeSeriesList;
+  let datapointArray: DatapointsPostDatapoint[] = [];
+  let timeseriesArray: PostTimeSeriesMetadataDTO[] = [];
+  beforeAll(async () => {
+    client = setupLoggedInClient();
+    axiosMock = new MockAdapter(client.instance);
+
+    // Create timeseries
+    timeseriesArray = [
+      {
+        name: 'test-timeseries',
+        externalId: 'timeseries' + randomInt(),
+      },
+    ];
+    for (let index = 0; index < 2; index++) {
+      timeseriesArray.push({
+        name: 'test-timeseries' + randomInt(),
+        assetId: randomInt(),
+      });
+    }
+    axiosMock
+      .onPost(new RegExp('/timeseries$'), {
+        items: timeseriesArray,
+      })
+      .replyOnce(200, {
+        items: timeseriesArray.map(timeseries => ({
+          ...timeseries,
+          id: randomInt(),
+        })),
+      });
+    createdTimeSeries = await client.timeseries.create(timeseriesArray);
+
+    // Create datapoints
+    datapointArray = [];
```

Comment:
I don't see why you need to mock etc to create this array. You can just manually define the datapointArray?